### ISSUE TITLE
Unclear element for setting up USB installer

### ISSUE THREAD

**Issue by ThSGM** [swanux/t440p#4]
Hi! Can I clarify these instructions? This is for the debug version.

`9. Choose option 2, then drag the Config.plist from the EFI folder into the terminal.`

Here, do you mean that we should drag `Versions/Debug/EFI/OC/Config.plist` into the terminal when prompted?

`11. Copy EFI and SoundFix.zip folders to USB_EFI (if you're using EFI_DEBUG rename it to EFI after copying to USB, and copy original EFI as EFI_NORMAL or something like this)`

Here, do you mean that we should then drag the folder `Versions/Debug/EFI` into the the EFI disk on the USB drive? So within the EFI disk, there is a separate EFI folder? 

I was not sure what you meant by renaming, since the directories are named EFI in your repository.

I thought I made the right USB installer but apparently not. I got the following error when I tried to boot to the disk:

![Screenshot 2020-10-11 at 01 25 33](https://user-images.githubusercontent.com/3396578/95667638-b5375380-0b60-11eb-9c4e-f13205224bb1.png)

**Comment by swanux** [comment]
Hi,
first of all, thank you for the feedback. Excuse me for the inconvenience, it seems that I forgot to rewrite the Readme according to the new file structure (as the Readme is written for the released version, not the most recent one) 

I'm going to adjust the instructions within the next few hours. If you have questions afterwards, feel free to ask. 

EDIT:
I've updated the repo accordingly.

**Comment by ThSGM** [comment]
Hi. Thanks! I'm redoing the install now. 

Just a question. On my machine, I had a failed install of Clover. [More info here.](https://www.reddit.com/r/hackintosh/comments/j8skep/kernel_panic_why_do_the_boot_options_remain/)

Should I need to remove these boot options or reset the NVRAM or something before attempting your install? Can you walk me through how to reset the NVRAM? I was a bit scared off because in the guide by Sniki, it is written: 

`RESET NVRAM: please do not use this option, there is an issue from Lenovo's side on the 2-3 recent versions of Bios which after NVRAM reset, the Laptop get's bricked and you will have to flash original bios back with a SPI Programmer.`

**Edit** I can confirm I get exactly the same error as in my original post after doing the USB installation disk again.

**Comment by swanux** [comment]
I never had to reset NVRAM with macOS in single-boot mode. I guess it should work without Windows installed. Maybe you could try to boot the USB without the Windows disk attached.
**But** first of all, maybe you could try a non-debug EFI (USBTethering for example), as it may be a bug in the DEBUG binaries according to [this](https://www.insanelymac.com/forum/topic/338516-opencore-discussion/?do=findComment&comment=2693716)

**Comment by ThSGM** [comment]
This is great stuff. I managed to get it all installed following your directions and and using the non-debug EFI (USBTethering). It was a bit unclear to me where network connectivity was used during the install but nevertheless I tethered my phone to it. I can confirm that the bug you linked seems real. 

Some comments maybe to help you with revision:

> Copy EFI and SoundFix.zip folders to USB_EFI

I would rephrase to:

> Copy EFI **folder** and SoundFix.zip **file** to the root directory of USB_EFI`

Unless you really did mean **folder**. Should we be unzipping SoundFix?

Item 10, 

> Alt+Space again, and open terminal. Type sudo diskutil mountDisk disk0. (In case your disk isn't disk0, replace it with correct value)

I would rephrase to: 

> Alt+Space again, and open terminal. Type sudo diskutil mountDisk **disk0s1. (where disk0s1 corresponds to the EFI partition of the main disk with the new Mac install)**

Also, I would rephrase

> Open Finder and copy EFI folder from USB_EFI partition to EFI partition. Note If you used EFI_DEBUG, then do not copy that, copy the other one, EFI_NORMAL for example

to 

> Open Finder and copy EFI folder from USB_EFI partition **to the main disk** EFI partition. **Make sure you copy the same EFI folder from the version you used in creating the bootable disk.** 

For those Mac users, you may want to advise the command `diskutil list` to list the volumes.

I know that many of these are obvious to most users, but there is no harm in being precise.

**Comment by swanux** [comment]
I'm glad that it worked! I'll replace the debug EFI once I update the whole guide to OC 0.6.2 and Big Sur. (probably in novembre)

Regarding network connectivity, it's needed when creating usb from non Mac computer, as it downloads the system during install. 

Thank you for your suggestions! As I'm more interested in developing stuff than writing / explaining things, I can be really terrible / confusing when I'm trying to do so. So I really appreciate your corrections, and I'll implement them the next weekend.

**Comment by ThSGM** [comment]
Thanks for all your help.

Just to verify, what is the deal with SoundFix.zip? Should this be unzipped into the EFI? What part of the process calls this and what does it do?

**Comment by swanux** [comment]
Sorry for the delay.

SoundFix is optional if you have issues with one ore more of the followings: strong white noise through jack after wake from sleep, need to manually switch audio outputs, no audio at all or no audio through headphones.

It can be extracted to the desktop for example and it has an installer script which you need to run. I've created it by slightly modifying AlcPlugFix/Audio Fix.
